packages/web/src/shared/accessControl.tsx

```diff
@@ -127,12 +127,15 @@ export interface UserCanOptions {
  */
 export function useAccessControl() {
   const { status, user } = useAuth();
-
+  // From https://stackoverflow.com/questions/59467758/passing-array-to-useeffect-dependency-list/59468261#59468261
+  // The following variable is used to update the policy when a user's languages change.
+  const languagesStringified = JSON.stringify(user?.languages);
   const policy = useMemo(() => {
     if (status !== 'loading') {
       return createPolicyFor(user);
     }
-  }, [status, user]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [status, user, languagesStringified]);
```

Comment:
How did you decide that we need to add this to the effect? Whenever the session is reloaded, `user` should be a brand new object which should already trigger this effect

Reply:
I overthought and undertested this. I was thinking it was possible that the user object might remain the same, but the translation roles might change. I added this line and then the call to `invalidateQueries` in `InviteLanguageMemberView` and came to the conclusion that they were both needed to work. 

Fixed.